### ISSUE TITLE
Update custom operator example

### ISSUE THREAD

**Issue by Airblader** [moroshko/rxviz#9]
I think the custom operator example should be rewritten to use pipeable operators instead of mutating the prototype:

```
const sqrt = source$ => Rx.Observable.create(observer =>
    source$.subscribe(
      value => {
        const result = Math.sqrt(value);

        if (typeof value !== 'number' || isNaN(result)) {
          observer.error(`Square root of ${value} doesn't exist`);
        } else {
          observer.next(result);
        }
      },
      err => observer.error(err),
      () => observer.complete()
    )
  );

Rx.Observable
  .interval(1000)
  .pipe(sqrt)

```

**Comment by moroshko** [comment]
Sounds good!
Could you submit a PR, please?

**Comment by moroshko** [comment]
Merged #10 and deployed.
